src/Model/Price/NullPrice.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace TeamGantt\Dues\Model\Price;
+
+use TeamGantt\Dues\Model\Price;
+
+class NullPrice extends Price
+{
+    public function __construct()
+    {
+        parent::__construct(0.0);
+    }
+
+    public function getAmount(): float
+    {
+        return -0.01;
```

Comment:
should this return `(float) 0` instead?

Reply:
![Screenshot_20201208_112506](https://user-images.githubusercontent.com/18272064/101510524-0d63b980-3948-11eb-8d35-a038e53efa17.png)

Reply:
I thought it might be a good idea to avoid `0` making it into any price 😱 . It shouldn't ever matter in the case of a `NullPrice`, but a negative value is probably more likely to trigger an error, and we would notice if we all of a sudden were shelling out pennies to everyone!

Reply:
I feel like there's a movie that has to do with this.. lol.